demo/pages/aggregation/index.html

```diff
@@ -5,9 +5,9 @@
 				<b>count: </b> {{$cell.value}}
 			</q-grid:template>
 		</q-grid:column>
-		<q-grid:column key="contact.address.zip" aggregation="avg"></q-grid:column>
+		<q-grid:column key="contact.region" aggregation="{type: 'avg', 'distinct': true}" ></q-grid:column>
```

Comment:
can we have aggreation ang aggregation-options as separate fields?
90% of aggregations will be simple cases, and we'are triing to have the same type for the model props